docs/xls-47d-price-oracles/transaction-types/oracleset.md

```diff
@@ -63,15 +63,14 @@ The oracle provider must complete these steps before submitting this transaction
 |---------------------|-----------|---------------|-----------|-------------|
 | `BaseAsset`         | String    | Currency      | Yes       | The primary asset in a trading pair. Any valid identifier, such as a stock symbol, bond CUSIP, or currency code is allowed. For example, in the BTC/USD pair, BTC is the base asset; in 912810RR9/BTC, 912810RR9 is the base asset. |
 | `QuoteAsset`        | String    | Currency      | Yes       | The quote asset in a trading pair. The quote asset denotes the price of one unit of the base asset. For example, in the BTC/USD pair, BTC is the base asset; in 912810RR9/BTC, 912810RR9 is the base asset. |
-| `AssetPrice`        | Number    | UInt64        | No        | The asset price after applying the `Scale` precision level. It's not included if the last update transaction didn't include the `BaseAsset`/`QuoteAsset` pair. |
-| `Scale`             | Number    | UInt8         | No        | The scaling factor to apply to an asset price. For example, if `Scale` is 6 and original price is 0.155, then the scaled price is 155000. Valid scale ranges are 0-10. It's not included if the last update transaction didn't include the `BaseAsset`/`QuoteAsset` pair.|
+| `AssetPrice`        | Number    | UInt64        | No        | The asset price after applying the `Scale` precision level. |
+| `Scale`             | Number    | UInt8         | No        | The scaling factor to apply to an asset price. For example, if `Scale` is 6 and original price is 0.155, then the scaled price is 155000. Valid scale ranges are 0-10. |
 
-`PriceData` is created or updated, following these rules:
+`PriceData` objects in the `PriceDataSeries` array are created, updated, or deleted following these rules:
 
-- New token pairs in the transaction are added to the object.
-- Token pairs in the transaction overwrite corresponding token pairs in the object.
-- Token pairs in the transaction with a missing `AssetPrice` field delete corresponding token pairs in the object.
-- Token pairs that only appear in the object have `AssetPrice` and `Scale` removed to signify that the price is outdated.
+- **Create**: Add a `PriceData` token pair that doesn't exist in the data series.
```

Comment:
It seems as though a `PriceData` object can contain multiple token pairs. If that's the case, I would suggest "Add a token pair that doesn't exist in the data series to the `PriceData` object," or some such clarification. As is, it sounds like `PriceData` is a token pair.